customer-bill/src/main/java/org/fiware/tmforum/customer_bill/domain/TimePeriod.java

```diff
@@ -0,0 +1,13 @@
+package org.fiware.tmforum.customer_bill.domain;
```

Comment:
Please no underscores in package names. I usually follow(and enforce;)) this guideline  https://google.github.io/styleguide/javaguide.html#s5.2.1-package-names